src/InteractiveWindow/EditorTest/InteractiveWindowTests.cs

```diff
@@ -764,6 +764,30 @@ public void Paste()
         }
 
         [WpfFact]
+        public void CopyAndPasteFullLineNoSelection()
```

Comment:
I think box selection has issues across all copy/paste related operations so it would be better to address them together, i will do it next.